src/Compilers/CSharp/Portable/Symbols/ConstructedNamedTypeSymbol.cs

```diff
@@ -24,7 +24,7 @@ internal SubstitutedNestedTypeSymbol(SubstitutedNamedTypeSymbol newContainer, Na
 
         internal override ImmutableArray<TypeSymbolWithAnnotations> TypeArgumentsNoUseSiteDiagnostics
         {
-            get { return TypeParameters.SelectAsArray(TypeMap.AsTypeSymbolWithAnnotations); }
```

Comment:
>TypeMap.AsTypeSymbolWithAnnotations [](start = 54, length = 35)

Just to confirm, `TypeMap.AsTypeSymbolWithAnnotations` is still used elsewhere in the code? Should it be used in `GetTypeParametersAsTypeArguments`?

Reply:
`AsTypeSymbolWithAnnotations` sets `IsNullable=false` for reference types while `GetTypeParametersAsTypeArguments` sets `IsNullable=false` if this module uses nullability and `IsNullable=null` otherwise.

---
In reply to: [171066926](https://github.com/dotnet/roslyn/pull/25087#discussion_r171066926) [](ancestors = 171066926)